source/core.py

```diff
@@ -424,7 +424,12 @@ def main():
 	"""
 	log.debug("Core starting")
 
-	ctypes.windll.user32.SetProcessDPIAware()
+	if winVersion.getWinVer() >= winVersion.WIN10_1703:
+		ctypes.windll.user32.SetProcessDpiAwarenessContext(-4)  # DPI_AWARENESS_CONTEXT_PER_MONITOR_AWARE_V2

```

Comment:
Could you define these constants at the top of the file, and include a link to documentation where they are defined.

```suggestion
# insert link to documentation for this
PROCESS_PER_MONITOR_DPI_AWARE = 2
DPI_AWARENESS_CONTEXT_PER_MONITOR_AWARE_V2 = -4
....
		ctypes.windll.user32.SetProcessDpiAwarenessContext(DPI_AWARENESS_CONTEXT_PER_MONITOR_AWARE_V2 )
```